src/cwv/handler.js

```diff
@@ -27,9 +31,19 @@ export async function CWVRunner(auditUrl, context, site) {
     interval: INTERVAL,
     granularity: 'hourly',
   };
-  const cwvData = await rumAPIClient.query('cwv', options);
+  const cwvData = await rumAPIClient.queryMulti(CWV_QUERIES, options);
   const auditResult = {
-    cwv: cwvData.filter((data) => data.pageviews >= DAILY_THRESHOLD * INTERVAL),
+    cwv: cwvData.cwv.filter((data) => data.pageviews >= DAILY_THRESHOLD * INTERVAL)
+      .map((cwvItem) => {
+        // Find a matching formVital by URL
+        const matchingFormVital = cwvData.formVitals.find(
+          (formVital) => formVital.url === cwvItem.url,
+        );
+
+        // eslint-disable-next-line max-len
+        const hasForm = matchingFormVital ? (matchingFormVital.isFormViewPresent || matchingFormVital.isFormSubmitPresent) : false;
```

Comment:
we should have a formElementClickPresent also in case view and submit are not able to detect the form.